src/TraceEvent/Computers/StartStopActivityComputer.cs

```diff
@@ -74,8 +74,8 @@ public StartStopActivityComputer(TraceLogEventSource source, ActivityComputer ta
 
             Dictionary<long, int> mapConnectionToTargetProcess = new Dictionary<long, int>();
 #endif 
-            var dynamicParser = source.Dynamic;
-            dynamicParser.All += delegate (TraceEvent data)
+            //var dynamicParser = source.Dynamic;
```

Comment:
I haven't figured out how to integrated the dynamic templates generated from EventPipeTraceEventParser to DynamicParser.  So I need to subscribe AllEvents to get EventSource events. That's also the reason I add EventPipeTraceEventParser in TraceLog.RegisterStandardParsers. Need suggestion.